src/engine/command.cpp

```diff
@@ -3579,11 +3579,24 @@ void getvarinfo(int n, int types, int notypes, int flags, int noflags, char *str
     }
     if(str && *str)
     {
+        std::vector<std::string> words;
+        explodelist(str, words);
         static char *laststr = NULL;
         if(ids[1].empty() || !laststr || strcmp(str, laststr))
         {
             ids[1].setsize(0);
-            loopv(ids[0]) if(rigcasestr(ids[0][i]->name, str)) ids[1].add(ids[0][i]);
+            loopv(ids[0])
```

Comment:
please do not use loop macros, instead use `for (int i = 0; i < ids[0].length(); i++)`

Reply:
I didn't add that loop macro. I think getting rid of the loop macros is a separate issue for another PR.

Reply:
I general we usually just remove them from code we rework/work at. It doesn't hurt anybody